### PR TITLE
String fixes

### PR DESCRIPTION
**Pull request by silopolis**
Small plurals in AdminSettings.vue component

### CODE REVIEW

File: src/components/AdminSettings.vue [hunk 1]
```diff
@@ -578,12 +578,12 @@ export default {
 				break
 			case 'other_error':
 				showError(
-					t('integration_openproject', 'Failed to revoke some user(s) OpenProject OAuth access token(s)')
+					t('integration_openproject', 'Failed to revoke some users\' OpenProject OAuth access tokens')
```
Comment (phil-davis) [inline src/components/AdminSettings.vue:581]: This is correct use of the apostrophe to indicate the possessive of a plural. There are "some users" (plural) and then we need to indicate the possessive of the plural "users" and so the apostrophe goes after the "s".

Ref: https://www.thepunctuationguide.com/apostrophe.html

**Review by phil-davis — APPROVED** [review]
LGTM

File: src/components/AdminSettings.vue (hunk 1 above)
Comment (SwikritiT) [reply src/components/AdminSettings.vue:581]: what about 'tokens'?If there's only one token/

Comment (phil-davis) [reply src/components/AdminSettings.vue:581]: > what about 'tokens' ? If there's only one token?

IMO that is OK grammar. Example: "I created some files" - when "some" is used then the noun is plural "files" Usually when the word "some" is used we think "more than one", but I suppose there could be only one in a particular case.

If it is often the case that there is just one access token that failed to be revoked, then I suppose the code could try to detect that, and write a complete sentence in the singular.

**Comment by SwikritiT** [comment]
Hi, @silopolis thank you for this PR. Some of the unit tests are failing because of this.
Can you please update with the same changes here as well
https://github.com/nextcloud/integration_openproject/blob/9f82042d7291b2120c8a7eeb78b5802afb14b70b/tests/jest/components/AdminSettings.spec.js#L953

https://github.com/nextcloud/integration_openproject/blob/9f82042d7291b2120c8a7eeb78b5802afb14b70b/tests/jest/components/AdminSettings.spec.js#L958

**Comment by silopolis** [comment]
Of course, will do later today 👍

**Comment by silopolis** [comment]
A bit late, but done :)

**Review by individual-it — APPROVED** [review]

**Comment by SwikritiT** [comment]
@individual-it maybe we can merge this? The tests are passing
https://github.com/nextcloud/integration_openproject/pull/381,
CI is red because of the coverage cannot be posted from a fork.

**Pull request merged**